app/docs/reference/compatible-devices.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Compatible devices | Reference"
+description: "Lists devices compatible with Routegy QR codes and NFC tags"
+sidebar: false
+---
+
+# Devices compatible with QR and NFC touchpoints
+
+Interacting with Routegy QR and NFC touchpoints relies on built-in capabilities of customer's device. These capabilities are now a standard feature in most modern devices that run either iOS or Android.
+
+
+## iOS device compatibility
+
+QR code reading capability is built into the camera app on any iPhone, iPad and iPod that runs iOS 11 or newer.
```

Comment:
I'd bold the version so it pops a bit more, e.g. **iOS 11**. Same below with **iOS 13** and the other places where you call the Android OS versions as well.